benchmarks/aggregate.py

```diff
@@ -297,11 +328,13 @@ def pr_latest(results_map: Dict[str, Any], args, timestamps: List[str]):
     logger.warning(f'cannot find data for accelerator {args.accelerator}')
     return
 
-  if args.format == 'csv':
-    print(','.join(['# Workload'] + [
-        f'Speedup({title}/{args.baseline.capitalize()} {base_backend_name}),StdDev,ModelName({title})'
-        for title in titles
-    ]))
+  if args.format == 'csv' or args.format == 'tab':
+    headers = ['Workload'] + [
+        header for title in titles for header in [
+            speedup_header(title, base_backend_name, args), 'StdDev',
+            modelname_header(title, args)
+        ]
+    ]
     # Note: the latest timestamp might not have results for all benchmarks.
     max_len = max([len(l) for l in speedups])
```

Comment:
`s/max_len/row_size`

?

Reply:
Ended up using `num_rows`, otherwise `row_size` might be confused with `num_cols` :-P

Reply:
Also added a comment explaining why we pad the lists.